### PR TITLE
Frontend 1872 - Use Synapse Typography/Styles Across Dev Center

### PR DESCRIPTION
**Pull request by langlk**
Resolves Dev Center side of [FRONTEND-1872](https://algorithmia.atlassian.net/browse/FRONTEND-1872)

**Notes:**
* This should fix all fonts/images, if you see images going past the column or the old fonts, that needs fixing
* Relies on styles in https://github.com/algorithmiaio/synapse/pull/184
* `/developers/getting-started` is broken currently, but since we're going to update the code samples to use Vue, I'm going to leave as is till we can switch it to use those.

### CODE REVIEW

**Comment by bnlb** [comment]
What do you think about updating this element (`_layouts/synapse-default.html`):

```
          <div class="syn-col syn-col-lg-7">
            {{ content | emojify }}
          </div>
```
so that it uses `syn-col-lg-8`? The code snippet on /developers is getting clipped. In prod, the main content (`#wrapper > div.container.dev-center-content-container > div > div.col-sm-8.col-md-8.col-sm-push-3.col-md-push-3`) is 755px wide, whereas on localhost it's currently 719px. Are there any issues with bumping the size up just a tad?

Screen Shot 2019-11-05 at 12 14 28 PM

**Comment by bnlb** [comment]
http://localhost:4000/developers/algorithm-development/algorithm-checklist/

The text on this page is a bit cramped. It looks like we need `.syn-body-1` on all the `<p>` tags?
Screen Shot 2019-11-05 at 12 24 00 PM

**Comment by langlk** [comment]
> What do you think about updating this element (`_layouts/synapse-default.html`):
> 
> ```
>           <div class="syn-col syn-col-lg-7">
>             {{ content | emojify }}
>           </div>
> ```
> 
> so that it uses `syn-col-lg-8`? The code snippet on /developers is getting clipped. In prod, the main content (`#wrapper > div.container.dev-center-content-container > div > div.col-sm-8.col-md-8.col-sm-push-3.col-md-push-3`) is 755px wide, whereas on localhost it's currently 719px. Are there any issues with bumping the size up just a tad?
> 
> Screen Shot 2019-11-05 at 12 14 28 PM

I think it's pretty likely the two-column formatting inside the main content is going to go away, as per [this abstract spec](https://app.goabstract.com/projects/45ebab60-6b71-11e9-bf8b-7f34265f9d6c/branches/master/commits/1cc9136a1cd270cd28b66cc117d7562e3a9d8c0c/files/853AC8A9-31A5-4140-80AF-E16B6D1CE519/layers/67E818AA-AE7F-4477-8133-D3747FE46593?mode=design).

**Comment by langlk** [comment]
> http://localhost:4000/developers/algorithm-development/algorithm-checklist/
> 
> The text on this page is a bit cramped. It looks like we need `.syn-body-1` on all the `<p>` tags?
> Screen Shot 2019-11-05 at 12 24 00 PM

Updated! Thanks!

File: _pages/faqs.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ redirect_from:
 ---
 
 <div id="faqs-index">
-  <section class="row row-sm">
-    <div class="col-md-6">
+  <div class="syn-row syn-mb-32">
```
Comment (bnlb) [inline _pages/faqs.md]: Why the move away from `<section>`?

Comment (langlk) [reply _pages/faqs.md]: I just feel better using `div`s for things, since they don't come with any additional styling

**Pull request merged**